app/components/search-result/template.hbs

```diff
@@ -110,26 +118,22 @@
     </div>
 {{/if}}
 <div class="row extra">
-    <div class="col-xs-8">
+    <div class="col-xs-3">
```

Comment:
This could be changed to 6 to give the updated date more space:
![screen shot 2016-08-15 at 2 09 03 pm](https://cloud.githubusercontent.com/assets/7131985/17674083/077346b6-62f2-11e6-8565-592ae3af5677.png)